scylla/tests/integration/compression.rs

```diff
@@ -0,0 +1,114 @@
+use crate::utils::{test_with_3_node_cluster, unique_keyspace_name, PerformDDL};
+use scylla::client::session::Session;
+use scylla::client::session_builder::SessionBuilder;
+use scylla::statement::query::Query;
+use scylla::frame::Compression;
+
+use scylla_proxy::{ProxyError, WorkerError, ShardAwareness, RequestRule, Reaction, RequestReaction, Condition, RequestOpcode};
+
+use tokio::sync::mpsc;
+use std::sync::Arc;
+
+async fn test_compression(compression: Option<Compression>, text_size: usize, expected_frame_total_size_range: std::ops::Range<usize>) {
```

Comment:
:wrench: Could you please add a docstring / comment explaining the semantics of all parameters and the logic inside?

Reply:
add a docstring (I think it's in the right convention)